recipes/urwid/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set version = "1.3.1" %}
+
+package:
+  name: urwid
+  version: "{{ version }}"
+
+source:
+  fn: urwid-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/u/urwid/urwid-{{ version }}.tar.gz
+  md5: 2e1a005cb31368fe21bfeba2d6ad5a5c
+
+build:
+  number: 0
```

Comment:
Simply add the following after the build `number` in the `build` section.

    skip: true  # [win]